features/user_can_search_restaurant_assortment.feature

```diff
@@ -0,0 +1,17 @@
+Feature: visitor search for restaurant assortment
+  As a visitor
+  In order to get suggestion for a restaurant
+  I would like to be able to enter a keyword to find an assortment of restaurants to select from
+
+  Background:
+    Given the following restaurants exists
+      | name            | description                                      | assortment |
+      | ThaiTanic       | Thailands finest food, watch out for the iceberg | Thai       |
+      | Fu King Chinese | Best chinese food in town                        | Asian      |
+    And I visit the landing page
+
+  Scenario: Visitor can search for restaurants
+    And I fill in "assortment" with "Thai"
```

Comment:
To make this even more clear we could write: 
When I fill in the "assortment" field with "Thai"

Reply:
Ok, easy fix